### PR TITLE
Features/update priority types #161762862

### PR DESCRIPTION
**Pull request by akegan**
https://www.pivotaltracker.com/story/show/161762862

### CODE REVIEW

File: spec/javascript/components/applications/__snapshots__/ApplicationPage.test.js.snap [hunk 1]
```diff
@@ -557,7 +557,7 @@ Array [
                     Has ADA Priorities Selected
                   </h4>
                   <p>
-                    Vision impaired;Mobility impaired;Hearing impaired
+                    Vision impairments;Mobility impairments;Hearing impairments
```
Comment (callaghanc) [inline spec/javascript/components/applications/__snapshots__/ApplicationPage.test.js.snap:560]: @akegan This is fine in that it is the direct value from Salesforce and it matches the formatting that already existed here, but perhaps we can add spaces after the colons to make it look a little nicer and be easier to read?

Comment (akegan) [reply spec/javascript/components/applications/__snapshots__/ApplicationPage.test.js.snap:560]: I think there's a ticket in the backlog already to make all the formatted stuff look nicer in the application edit page

Comment (callaghanc) [reply spec/javascript/components/applications/__snapshots__/ApplicationPage.test.js.snap:560]: is this one you're thinking of? https://www.pivotaltracker.com/story/show/162027506

Comment (akegan) [reply spec/javascript/components/applications/__snapshots__/ApplicationPage.test.js.snap:560]: this one https://www.pivotaltracker.com/n/projects/1405352/stories/162028377.

Comment (callaghanc) [reply spec/javascript/components/applications/__snapshots__/ApplicationPage.test.js.snap:560]: ok, well looks like both of those cover different aspects of app page formatting, so seems like we've already planned to do this at a later time 👍

**Review by callaghanc — APPROVED** [review]

File: spec/javascript/components/applications/__snapshots__/ApplicationPage.test.js.snap (hunk 1 above)
Comment (akegan) [reply spec/javascript/components/applications/__snapshots__/ApplicationPage.test.js.snap:560]: oh whoops. didn't see ADA priorities in the other one.

**Pull request merged**